.github/workflows/normalize_repos.yml

```diff
@@ -0,0 +1,31 @@
+name: Normalize Repos
+
+on: 
+  schedule:
+  - cron: '0 0 * * *'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
```

Comment:
We don't need to run this across a bunch of different Python versions (which is what `matrix` does, I think). We just need to run once.

Reply:
Which version should I define this to then? Is there any preference?

Reply:
Python 3.7 should be best, see [normalize_repo's Pipfile](https://github.com/creativecommons/ccos-scripts/blob/master/normalize_repos/Pipfile#L12).

Reply:
Even I thought the same so I have made the changes in the latest commit

Reply:
We should remove the `matrix` command since it's just one version. See the "Basic" example in the readme of https://github.com/actions/setup-python for how to set up Python without the `matrix` command.

Reply:
Resolved in the latest commit.